en-US/Players_Guide/03_Creating_a_Character/Using_Ability_Scores/Proficiency_Bonus.md

```diff
@@ -0,0 +1,20 @@
+#### Proficiency Bonus
+
+Characters and monsters have a proficiency bonus determined by level, which is incorporated into their stat blocks and used in ability checks, saving throws, and attack rolls.
+
+Your proficiency bonus can’t be added to a single die roll or other number more than once.
+For example, if two different rules allow you to add your proficiency bonus to a Wisdom saving throw, you add it only once when you make the save.
+
+##### Expertise
```

Comment:
Note: I made this a subsection of the **Proficiency Bonus** section, since Expertise builds upon the proficiency bonus.
This is a change compared to the Google Doc, but similar to how it's done in the WotC SRD.
What do you think?